src/components/Buttons.js

```diff
@@ -15,7 +15,10 @@ function Buttons({ id }) {
         <button
           type="button"
           id="remove"
-          onClick={() => dispatch(removeBook(id))}
+          onClick={() => {
+            dispatch(removeBook(id));
+            dispatch(removeBookApi(id));
+          }}
         >
```

Comment:
- [ ] Currently, when I tried to remove a book, the book is removed from the API but this isn't reflected in the DOM. In other words, the book doesn't get removed from the page but when I refresh the page, then it's removed. Kindly fix this behavior.